CSharpSourceCode/Utilities/Extensions/AgentExtensions.cs

```diff
@@ -349,45 +355,51 @@ public static void ApplyDamage(this Agent agent, int damageAmount, Agent damager
             try
             {
                 // Registering a blow causes the agent to react/stagger. Manipulate health directly if the damage won't kill the agent.
-                if (agent.State == AgentState.Active || agent.State == AgentState.Routed)
```

Comment:
why remove the routed? don't we want to keep applying statuseffect damage to agents who are currently fleeing? Don't agree with this change either.

Reply:
I think this was an artefact of furious testing, I didn't mean to remove that logic :D